src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/respondent/RespondentForceCaseProgressionPersonalisation.java

```diff
@@ -0,0 +1,63 @@
+
+import static java.util.Objects.requireNonNull;
+import static uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCaseDefinition.APPELLANT_FAMILY_NAME;
+import static uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCaseDefinition.APPELLANT_GIVEN_NAMES;
+import static uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCaseDefinition.HOME_OFFICE_REFERENCE_NUMBER;
+import static uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCaseDefinition.LEGAL_REP_REFERENCE_NUMBER;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Set;
+import javax.validation.constraints.NotNull;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCase;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCaseDefinition;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.personalisation.EmailNotificationPersonalisation;
+
+@Service
+public class RespondentForceCaseProgressionPersonalisation implements EmailNotificationPersonalisation {
+
+    private final String respondentEmailTemplateId;
+    private final String respondentEmailAddress;
+
+    public RespondentForceCaseProgressionPersonalisation(
+        @NotNull(message = "appealDocumentDeletedTemplateId cannot be null")
+        @Value("${govnotify.template.forceCaseProgression.awaitingRespondentEvidence.to.caseBuilding.respondent.email}") String respondentEmailTemplateId,
+        @Value("${respondentEmailAddresses.respondentReviewDirection}") String respondentEmailAddress
+    ) {
+
+        this.respondentEmailTemplateId = respondentEmailTemplateId;
+        this.respondentEmailAddress = respondentEmailAddress;
+    }
+
+    @Override
+    public String getTemplateId() {
+        return respondentEmailTemplateId;
+    }
+
+    @Override
+    public Set<String> getRecipientsList(AsylumCase asylumCase) {
+        return Collections.singleton(respondentEmailAddress);
+    }
+
+    @Override
+    public String getReferenceId(Long caseId) {
+        return caseId + "_RESPONDENT_FORCE_CASE_PROGRESSION";
+    }
+
+    @Override
+    public Map<String, String> getPersonalisation(AsylumCase asylumCase) {
+        requireNonNull(asylumCase, "asylumCase must not be null");
+        return ImmutableMap.<String, String>builder()
+            .put("appealReferenceNumber", asylumCase.read(
+                AsylumCaseDefinition.APPEAL_REFERENCE_NUMBER, String.class).orElse(""))
+            .put("appellantGivenNames", asylumCase.read(APPELLANT_GIVEN_NAMES, String.class).orElse(""))
+            .put("appellantFamilyName", asylumCase.read(APPELLANT_FAMILY_NAME, String.class).orElse(""))
+            .put("legalRepReferenceNumber", asylumCase.read(LEGAL_REP_REFERENCE_NUMBER, String.class).orElse(""))
```

Comment:
Does the respondent need the `legalRepReferenceNumber` sent to them? What is the point?